src/Node.ts

```diff
@@ -3193,7 +3193,7 @@ addGetterSetter(Node, 'listening', true, getBooleanValidator());
 
 addGetterSetter(Node, 'preventDefault', true, getBooleanValidator());
 
-addGetterSetter(Node, 'filters', null, function (this: Node, val) {
+addGetterSetter(Node, 'filters', undefined, function (this: Node, val) {
```

Comment:
This is a subtle API change.

Perhaps we should widen the type of `filters` in `NodeConfig` to `filters?: Array<Filter> | null`?